#! /bin/sh /usr/share/dpatch/dpatch-run
# 25_bscan-v3-and-missing-opts.dpatch
#
# Description: adds missing and new options in the bscan utility's manpage. Also
#  introduces better consistency between the Bacula utilities manpages.
# Origin: vendor, 
#  http://bugs.debian.org/cgi-bin/bugreport.cgi?msg=45;filename=manpage-bscan-v3-and-missing-opts.patch;att=2;bug=499619
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=499619
# Forwarded: http://sourceforge.net/mailarchive/message.php?msg_name=200912021125.44398.kern%40sibbald.com
# Author: Lucas B. Cohen <lbc@members.fsf.org>
# Reviewed-by: Kern Sibbald <kern@sibbald.com>
# Last-Update: 2009.12.3
#
# 2009.12.7 : tweaked patch to a patchlevel 2, to accomodate dpatch and git
#
# created by Lucas B. Cohen <lbc@members.fsf.org> with dpatch v2.0.31

@DPATCH@

diff -urNad a/bacula-3.0.3/manpages/bscan.8 b/bacula-3.0.3/manpages/bscan.8
--- a/bacula-3.0.3/manpages/bscan.8	2009-12-03 12:44:26.000000000 +0100
+++ b/bacula-3.0.3/manpages/bscan.8	2009-12-03 12:45:14.000000000 +0100
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH BSCAN 8 "26 May 2006" "Kern Sibbald" "Network backup, recovery and verification"
+.TH BSCAN 8 "26 November 2009" "Kern Sibbald" "Network backup, recovery and verification"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .SH NAME
@@ -49,9 +49,15 @@
 .BI \-d\  nn
 Set debug level to \fInn\fP.
 .TP
+.BI \-dt
+Print timestamp in debug output.
+.TP
 .B \-m
 Update media info in database.
 .TP
+.B \-D
+Specify the driver database name (default: \fINULL\fP)
+.TP
 .BI \-n\  name
 Specify the database name (default: \fIbacula\fP)
 .TP
@@ -64,18 +70,24 @@
 .BI \-h\  host
 Specify database host (default: \fINULL\fP)
 .TP
+.BI \-t\  port
+Specify database port (default: 0)
+.TP
+.B \-p
+Proceed inspite of I/O errors.
+.TP
 .B \-r
 List records.
 .TP
 .B \-s
 Synchronize or store in Database.
 .TP
+.B \-S
+Show scan progress periodically.
+.TP
 .B \-v
 Verbose output mode.
 .TP
-.B \-p
-Proceed inspite of I/O errors
-.TP
 .BI \-V\  volume
 Specify volume names (separated by '|')
 .TP
